pkg/jobmanager/jobmanager.go

```diff
@@ -182,6 +183,7 @@ loop:
 	}
 	// Stop the API (if not already)
 	jm.StopAPI()
+	<-errCh
```

Comment:
This effectively waits for API shutdown, is that right?

Reply:
correct